runtime/runtime-params-estimator/src/cost.rs

```diff
@@ -79,10 +79,24 @@ pub enum Cost {
     /// a transaction. Subtract base costs and apply least-squares on the
     /// results to find the per-byte costs.
     ActionDeployContractPerByte,
+    /// Estimates `action_creation_config.function_call_cost`, which is the base
+    /// cost for adding a `FunctionCallAction` to a receipt. It aims to account
+    /// for all costs of calling a function that are already known on the caller
+    /// side.
+    ///
+    /// Estimation: Measure the cost to execute a transaction with an empty
+    /// function with no arguments. Subtract the receipt creating cost from
+    /// that, as that is already charged separately.
```

Comment:
So this *would* include a cost of actually executing the function, right? I wonder if we can abuse our unncessary early bail-out for empty method names for estimation?

Reply:
Yes, it includes the execution of the empty function. I guess now that we are properly estimating the loading cost separately, this is an option. Let me see if it makes a difference.

Reply:
It does seem to make a sizeable difference from my quick hacky tests I did. But I think implementing that is asks for a separate PR, it's more than just a trivial change in the existing function.